python_modules/dagster/dagster/_core/storage/schedules/sql_schedule_storage.py

```diff
@@ -511,6 +511,17 @@ def get_auto_materialize_asset_evaluations(
             rows = conn.execute(query)
             return [AutoMaterializeAssetEvaluationRecord.from_db_row(row) for row in rows]
 
+    def purge_evaluations(self, before: float):
```

Comment:
should this be called purge_asset_evaluations?

Reply:
sure